Fix feature documentations (#325)
inveniosoftware/docs-invenio-rdm: docs/features/ux.md

![Creator dialog.](imgs/affiliations.png)

> Our auto-completion serves not only to make it easier for users, but also to ensure we obtain high-quality metadata and it can link to authority records.

Reviewer: ```suggestion
Our auto-completion makes it easier for users to deposit, but also ensures we obtain high-quality metadata. It can even be linked to authority records.
```
